app/models/pet_task.rb

```diff
@@ -0,0 +1,22 @@
+# A plain object that creates the default pet tasks for newly created Pet
+class PetTask
```

Comment:
This is where I would put this as a service instead of a model. This is the type of class that is meant to do one type action. So like a interactor or service object imo

Reply:
Yeh I am fine as a Service also. It's not what I would call domain logic.